.gitignore

```diff
@@ -12,3 +12,5 @@ npm-debug.log
 
 # Firebase
 firebase-debug.log
+
+.idea/
```

Comment:
IDE settings are usually configured in a global .gitignore as described here:

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

See for example [Global/JetBrains.gitignore](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore)